src/jitlayers.cpp

```diff
@@ -489,8 +501,13 @@ JuliaOJIT::JuliaOJIT(TargetMachine &TM)
         addOptimizationPasses(&PM);
     }
     else {
```

Comment:
could we write this as `addOptimizationPasses(&PM, /* -O */0)`, and make the `jl_options.opt_level` option to that function explicit?